hydroflow/examples/kvs_bench/server.rs

```diff
@@ -263,6 +263,10 @@ pub fn run_server<RX>(
 
                 // Send get results back to user
                 lookup
+                    -> map(|singleton_map: lattices::map_union::MapUnionSingletonMap<_, lattices::Pair::<MyLastWriteWins<BUFFER_SIZE>, MySetUnion>>| {
+                        let lattices::collections::SingletonMap(k, v) = singleton_map.into_reveal();
+                        (k, (v.into_reveal()))
+                    })
```

Comment:
I think I can make a recursive reveal to make this kind of thing easier

Reply:
shall I wait for you and add to this PR?

Reply:
decided not to